storage.bs

```diff
@@ -14,6 +14,10 @@ urlPrefix: https://tc39.github.io/ecma262/; spec: ECMASCRIPT
     text: agent cluster; url: #sec-agent-clusters; type: dfn
 </pre>
 
+<pre class=link-defaults>
+spec:infra; type:dfn; text:implementation-defined
```

Comment:
I think this might not be necessary after some changes made late in the day on Friday.

Reply:
Seems to still be necessary unfortunately.